pkg/drift_client/drift_client.py

```diff
@@ -77,6 +79,10 @@ def get_topics(self) -> List[str]:
         topics = self.__influx_client.query_measurements()
         return topics
 
+    @deprecation.deprecated(
```

Comment:
deprected_in= 0.2.0 - current version
removed_in=1.9.0 - version where we remove it